qa/tasks/cephfs/test_layout.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+class TestInheritedLayout(CephFSTestCase):
+
+    def test_layout_root(self):
+        self.mount_a.run_shell(["mkdir", "a"])
```

Comment:
Let's use `run_shell_payload` as the interface is much more friendly.

Reply:
Ack

Reply:
?